app/config/collections.php

```diff
@@ -3194,6 +3194,97 @@
             ]
         ],
     ],
+
+    'fileTokens' => [
```

Comment:
Didn't we discuss the potential to make this a global feature at some point? Can we make the collection more general or use the existing structure for collections?